### PR TITLE
Let users disable `hide-watch-and-fork-count`

### LINKED ISSUE #1843: Why you hide count field for "Watch" and "Fork" buttons?
**GitHub**

github-original

**GitHub refined**

github-refined

---

I personally prefer original variant.

If you don't want to fix it, maybe you have a CSS snippet to got it back?

### PR DESCRIPTION
**Pull request by DanSnow**
I add a new class `hide-watch-and-fork-count`.
Or maybe should I just use `btn.style.display = 'none'`?

Closes #1843

### CODE REVIEW

File: source/features/hide-watch-and-fork-count.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import features from '../libs/features';
+import select from 'select-dom';
+
+function init() {
+	for (const btn of select.all('.social-count[href$="/network/members"], .social-count[href$="/watchers"]')) {
```
Comment (bfred-it) [inline source/features/hide-watch-and-fork-count.tsx]: Like for https://github.com/sindresorhus/refined-github/blob/master/source/features/hide-useless-newsfeed-events.tsx, the class can just be added to `body` so we don't have to re-select the items in Javascript.

File: source/features/hide-watch-and-fork-count.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+		btn.classList.add('rgh-hide-watch-and-fork-count');
+	}
+}
+
+features.add({
+	id: 'hide-watch-and-fork-count',
```
Comment (bfred-it) [inline source/features/hide-watch-and-fork-count.tsx]: This needs:

```js
include: [
	features.isRepo
}
```

**Comment by DanSnow** [comment]
@bfred-it Thanks for reviewing. I have updated it. Can you please check again?

**Comment by bfred-it** [comment]
Thanks @DanSnow 🍴

**Comment by FatOrangutan** [comment]
.btn-with-count[title^='Fork your own'],
    [action='/notifications/subscribe'] .btn-with-count {
        border-radius: 0.25em; /* Matches GitHub's default */
    }

should have its selectors changed too. https://github.com/sindresorhus/refined-github/commit/adb319bf0201276ff73f43b3326a32ac16721df5

**Comment by bfred-it** [comment]
Fixed: 0c35487

**Comment by RichardLitt** [comment]
Thank you, all, for enabling this PR. I really appreciated your work here. <3

**Comment by hugovk** [comment]
This is now released in https://github.com/sindresorhus/refined-github/issues/1137#issuecomment-478207420.

To disable:

* Right-click the extension > Options > Features to disable > add `hide-watch-and-fork-count`

Thank you!

**Pull request merged**